tests/lib/snaps/test-snapd-rsync/snapcraft.yaml

```diff
@@ -0,0 +1,16 @@
+name: test-snapd-rsync
```

Comment:
@popey adopt a snap?

Reply:
Que?

Reply:
@sergiocazzolato packaged rsync as a snap as it's not currently present in the store but we needed it on Ubuntu Core system. Since rsync is super useful, I'm wondering if it makes sense to have an official rsync snap provided by snapcrafters instead (or next to) the one we have here.

Reply:
@bboozzoo Ok, I understand now, great idea.